docs/en/ingest-management/fleet/fleet-settings-output-kafka.asciidoc

```diff
@@ -0,0 +1,367 @@
+[[kafka-output-settings]]
+= Kafka output settings
+
+Specify these settings to send data over a secure connection to Kafka. 
+
+preview::[]
+
+[discrete]
+== General settings
+
+[cols="2*<a"]
+|===
+|
+[id="kafka-output-version"]
+**Kafka version**
+
+| The Kafka protocol version that {agent} will request when connecting. 
+Defaults to 1.0.0.
```

Comment:
Perhaps it's worth noting that we provide support for versions ranging from 0.8.2.0 to 2.6.0. However, it's important to highlight that the latest Kafka version (3.x.x) is expected to be compatible when using the version 2.6.0 setting. This is my understanding of the discussion on this matter; perhaps @faec can confirm.

Reply:
Added
>Currently Kafka versions from `0.8.2.0` to `2.6.0` are supported, however the latest 
>Kafka version (`3.x.x`) is expected to be compatible when version `2.6.0` is selected.